### PR TITLE
[13.0] Add project_sale_link

### PR DESCRIPTION
**Pull request by adrienpeiffer**
This module adds links to sales orders and customer invoices on projects

### CODE REVIEW

File: project_sale_link/models/project_project.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        return action
+
+    def _get_sale_order_domain(self):
+        return [("project_id", "in", self.ids)]
+
+    def _compute_customer_invoice_count(self):
+        for project in self:
+            domain = project._get_sale_invoice_line_domain()
+            groups = self.env["account.move.line"].read_group(
+                domain, ["move_id"], ["move_id"],
```
Comment (acsonefho) [inline project_sale_link/models/project_project.py]: Not possible to do this `read_group(...)` before the loop?
You group by `analytic_account_id` then you `sum(...)` results?

**Comment by adrienpeiffer** [comment]
@acsonefho Done :small_airplane:

File: project_sale_link/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+{
+    "name": "Project Sale Link",
+    "summary": """
+        Add sale and sale invoice link in project""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "depends": ["sale_timesheet", "project", "account"],
```
Comment (rousseldenis) [inline project_sale_link/__manifest__.py:12]: @adrienpeiffer Is sale_timesheet dependency really needed ?

**Review by acsonefho — APPROVED** [review]
Code LGTM

File: project_sale_link/__manifest__.py (hunk 2 above)
Comment (adrienpeiffer) [reply project_sale_link/__manifest__.py:12]: AS project is added by sale_timesheet, yes

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**